packages/contracts/contracts/interfaces/IMarginlyPool.sol

```diff
@@ -14,41 +15,68 @@ interface IMarginlyPool is IMarginlyPoolOwnerActions {
+  event DepositBase(address indexed user, uint256 amount, uint256 baseDiscountedDelta);
 
   /// @dev Emitted when user deposited quote token
   /// @param user Depositor
   /// @param amount Amount of token user deposited
-  event DepositQuote(address indexed user, uint256 amount);
+  /// @param quoteDiscountedDelta Discounted delta amount of quote tokens user deposited
+  event DepositQuote(address indexed user, uint256 amount, uint256 quoteDiscountedDelta);
 
   /// @dev Emitted when user withdrew base token
   /// @param user User
   /// @param amount Amount of token user withdrew
-  event WithdrawBase(address indexed user, uint256 amount);
+  /// @param baseDiscountedDelta Discounted delta amount of base tokens user withdrew
+  event WithdrawBase(address indexed user, uint256 amount, uint256 baseDiscountedDelta);
 
   /// @dev Emitted when user withdrew quote token
   /// @param user User
   /// @param amount Amount of token user withdrew
-  event WithdrawQuote(address indexed user, uint256 amount);
+  /// @param quoteDiscountedDelta Discounted delta amount of quote tokens user withdrew
+  event WithdrawQuote(address indexed user, uint256 amount, uint256 quoteDiscountedDelta);
 
   /// @dev Emitted when user shorted
   /// @param user Depositor
   /// @param amount Amount of token user deposited
   /// @param swapPriceX96 Price of swap worth in quote token as Q96
-  event Short(address indexed user, uint256 amount, uint256 swapPriceX96);
+  /// @param quoteDiscountedDelta Discounted delta amount of quote tokens
+  /// @param baseDiscountedDelta Discounted delta amount of base tokens
+  event Short(
+    address indexed user,
+    uint256 amount,
+    uint256 swapPriceX96,
+    uint256 quoteDiscountedDelta,
+    uint256 baseDiscountedDelta
+  );
 
   /// @dev Emitted when user made long position
   /// @param user User
   /// @param amount Amount of token user use in long position
   /// @param swapPriceX96 Price of swap worth in quote token as Q96
-  event Long(address indexed user, uint256 amount, uint256 swapPriceX96);
+  /// @param quoteDiscountedDelta Discounted delta amount of quote tokens
+  /// @param baseDiscountedDelta Discounted delta amount of base tokens
+  event Long(
+    address indexed user,
+    uint256 amount,
+    uint256 swapPriceX96,
+    uint256 quoteDiscountedDelta,
+    uint256 baseDiscountedDelta
+  );
 
   /// @dev Emitted when user closed position
   /// @param user User
   /// @param token Collateral token
   /// @param collateralDelta Amount of collateral reduction
   /// @param swapPriceX96 Price of swap worth in quote token as Q96
-  event ClosePosition(address indexed user, address indexed token, uint256 collateralDelta, uint256 swapPriceX96);
+  /// @param collateralDiscountedDelta Amount of discounted collateral reduction
+  event ClosePosition(
+    address indexed user,
+    address indexed token,
+    uint256 collateralDelta,
+    uint256 swapPriceX96,
+    uint256 collateralDiscountedDelta
```

Comment:
Either collateralDelta or collateralDiscountedDelta is definitely excessive here